src/Collapse/index.tsx

```diff
@@ -3,7 +3,28 @@ import {
   CollapseProps as AntdCollapseProps,
   CollapsePanelProps as AntdCollapsePanelProps,
 } from 'antd';
+import React, { ReactNode } from 'react';
 
 export const Collapse: typeof AntdCollapse = AntdCollapse;
 export type CollapseProps = AntdCollapseProps;
 export type CollapsePanelProps = AntdCollapsePanelProps;
+
+export function SingleCollapse(props: {
+  collapseProps?: Omit<CollapseProps, 'defaultActiveKey'> & {
+    defaultActive: boolean;
+  };
+  collapsePanelProps: Omit<CollapsePanelProps, 'key'>;
+  children: ReactNode;
+}) {
+  const key = 'default';
```

Comment:
Has to be reinstantiated on every rerender, should rather be an `ALL_CAPS` constant defined outside of the component.